### PR TITLE
Draw updatemenus over sliders

### PR DESCRIPTION
**Pull request by etpinard**
as pointed out by @jackparmer in https://plot.ly/~jackp/17246.embed

When a graph has both updatemenus and sliders, updatemenu buttons drop below sliders.
gifrecord_2017-01-13_174700

### CODE REVIEW

**Comment by etpinard** [comment]
I'll add a jasmine test about it next week. I just wanted to make a quick PR so that I didn't forget about it.

**Comment by rreusser** [comment]
Ah, I thought it would be a complicated SVG layering thing. Changing the order seems much simpler.

File: src/plot_api/plot_api.js [hunk 1]
```diff
@@ -196,8 +196,8 @@ Plotly.plot = function(gd, data, layout, config) {
 
         Registry.getComponentMethod('legend', 'draw')(gd);
         Registry.getComponentMethod('rangeselector', 'draw')(gd);
-        Registry.getComponentMethod('updatemenus', 'draw')(gd);
         Registry.getComponentMethod('sliders', 'draw')(gd);
+        Registry.getComponentMethod('updatemenus', 'draw')(gd);
```
Comment (rreusser) [inline src/plot_api/plot_api.js]: Real simple. The fact that it's quite this simple is a little concerning though.

- [Updatemenu is drawn on `_infolayer`](https://github.com/plotly/plotly.js/blob/master/src/components/updatemenus/draw.js#L55)
- [Slider is drawn on `_infolayer`](https://github.com/plotly/plotly.js/blob/master/src/components/sliders/draw.js#L29)

I guess that means it works. For some reason I thought the dropdown was drawn on a separate layer (should it be?). The only corner case I can think of:

1. Draw plot with updatemenus and no sliders
2. Add slider
3. Somehow the slider is appended *after* the updatemenu and ends up on top of it

But I guess that's what containerClass is for. So perhaps this is adequate.

Comment (rreusser) [inline src/plot_api/plot_api.js]: Ignore this. Seems all good. Had to mentally work through the hierarchy once more, but yeah, this seems good. I'll only add a small note about long-term robustness of relying upon component ordering. I can imagine needing `_upperinfolayer` or something to which expanded components are added. Not necessary now but might be a better solution if more cases arise.

Comment (etpinard) [inline src/plot_api/plot_api.js]: > Not necessary now but might be a better solution if more cases arise.

Yeah. Maybe the **best** solution would have the dropdown buttons in a separate layer above `<g class="infolayer">`. 

My solution aims to fix the 99% cases for users ASAP.

Comment (rreusser) [inline src/plot_api/plot_api.js]: Yup. That fix is nicely future-aware, but unless I'm wrong there's not actually any difference in the present.

**Pull request merged**